src/pages/alumni/index.js

```diff
@@ -31,58 +30,40 @@ class Alumni extends Component {
             </h1>
           </div>
         </header>
-        <div className="batch">Batch 2020</div>
+
+        <div className="batch">Batch 2021</div>
         <div className="grid md:grid-cols-2 gap-y-2 sm:grid-cols-1">
-          {this.state.dummy.map((test) => (
-            <Cards
+          {this.state.dummy.map((test, index) => (
+            test.batch == "2017" && <Cards
```

Comment:
There is an 'id' entry in backend.......you can use that for 'key' prop while mapping. Using 'name' is not a long-term solution.